packages/website/docs/FlatSelectConnected.md

```diff
@@ -0,0 +1,27 @@
+# Best Practices
+The Flat select option should be used for single choice selection.
+
+The width of each of the options should be the same, that way we keep everything equal and balanced, we don't want any of the options feel more important than the other.
+
+Limit the number of options, a good rule of thumb is to have no more than five.
+
+Don't use Flat select when the effect is not immediate, consider using dropdown menu, radio buttons or checkboxes instead. Users are expect to have immediate feedback on their actions when using Flat select.
+
+If used as view switcher (tabs), make sure it makes sense hierarchically and that tabs cannot be used.
+
+## Labeling
+- Keep titles short, preferably **under three words**.
+- Write for scannability
+
+## Flat select vs Toggle switch vs Tabs
+The Flat select share similarities with various other components but here we try to clarify the difference between some of them.
+
+## Flat select vs Tabs
+Both are similar in appearance and designed for view switching on an element. Flat select are suitable for view switching within a pane, as where tabs, are used as a navigational control (Moving from screen to another).
+Flat select vs Toggle switch
+Both are very similar but a key difference is that Toggles are for only two choices (on or off) and Flat select allows for multiple choices (previous, play, stop, next)
```

Comment:
We don't have a Toggle component on plasma AFAIK, we use the Flat select for that with true/false options or a checkbox. We probably should have a toggle I guess, but we currently don't

Reply:
I think we should until we do have the component, otherwise we advise doing something impossible